yarn.lock

```diff
@@ -2800,28 +2800,28 @@
   resolved "https://registry.yarnpkg.com/@popperjs/core/-/core-2.9.3.tgz#8b68da1ebd7fc603999cf6ebee34a4899a14b88e"
   integrity sha512-xDu17cEfh7Kid/d95kB6tZsLOmSWKCZKtprnhVepjsSaCij+lM3mItSJDuuHDMbCWTh8Ejmebwb+KONcCJ0eXQ==
 
-"@probe.gl/bench@3.5.0-alpha.4":
-  version "3.5.0-alpha.4"
-  resolved "https://registry.yarnpkg.com/@probe.gl/bench/-/bench-3.5.0-alpha.4.tgz#50573e2f8bd347ee3ff931619ce31df91fc24a53"
-  integrity sha512-N70nDI/Cd71Q9v9p8Dyli0totCcZAwG+GiAPMluRmYLO5L5bGts4loeSXlvsa0ylMvBFwqabGhNB+jE3xyF+ew==
+"@probe.gl/bench@^3.5.0":
+  version "3.5.0"
+  resolved "https://registry.yarnpkg.com/@probe.gl/bench/-/bench-3.5.0.tgz#54b0a3fe3912c4ad0f9c962daacf4a40f8c0075e"
+  integrity sha512-7E5UHxOAGxmwO446kpcVQeiXe+4ddODuFTgZ+svDkJMR3X2EltPuMsROfuKoSXz3RcLhoiyiK3PfvY65VvXGpQ==
   dependencies:
     "@babel/runtime" "^7.0.0"
-    "@probe.gl/log" "3.5.0-alpha.4"
+    "@probe.gl/log" "3.5.0"
 
-"@probe.gl/env@3.5.0-alpha.4", "@probe.gl/env@^3.5.0-alpha.4":
-  version "3.5.0-alpha.4"
-  resolved "https://registry.yarnpkg.com/@probe.gl/env/-/env-3.5.0-alpha.4.tgz#7b902a16eff84f67a729960a28cff75636c66c2f"
-  integrity sha512-8QWv17Cb0xMCXMMyrFyVO2Bz2JPWGl59TU6+nzAk+LASkR9+bVAXbHt9Iq3BxznItJUyFj6uJUo4hDlq3YT3pA==
+"@probe.gl/env@3.5.0", "@probe.gl/env@^3.5.0":
+  version "3.5.0"
+  resolved "https://registry.yarnpkg.com/@probe.gl/env/-/env-3.5.0.tgz#c6d8fbf414fda3eba7b3813cc274e76245216fe0"
+  integrity sha512-YdlpZZshhyYxvWDBmZ5RIW2pTR14Pw4p9czMlt/v7F6HbFzWfAdmH7q6xVwFRYxUpQLwhWensWyv4aFysiWl4g==
   dependencies:
     "@babel/runtime" "^7.0.0"
 
-"@probe.gl/log@3.5.0-alpha.4", "@probe.gl/log@^3.5.0-alpha.4":
-  version "3.5.0-alpha.4"
-  resolved "https://registry.yarnpkg.com/@probe.gl/log/-/log-3.5.0-alpha.4.tgz#0c7c2a1412e8d3641739d03b00fd8cd7bfc27ec3"
-  integrity sha512-bqrma+KDrfubmPznWEtHGfjECkqoNg1CCEQrfSFzqWswkvJ6s40zoxZGh44n0OUOzMon8O83pXxBK2SC8seFEg==
+"@probe.gl/log@3.5.0", "@probe.gl/log@^3.5.0":
+  version "3.5.0"
+  resolved "https://registry.yarnpkg.com/@probe.gl/log/-/log-3.5.0.tgz#6589822ab771eadf77787ffc6ecf73e59d181c64"
+  integrity sha512-nW/qz2X1xY08WU/TsmJP6/6IPNcaY5fS/vLjpC4ahJuE2Mezga4hGM/R2X5JWE/nkPc+BsC5GnAnD13rwAxS7g==
   dependencies:
     "@babel/runtime" "^7.0.0"
-    "@probe.gl/env" "3.5.0-alpha.4"
+    "@probe.gl/env" "3.5.0"
 
 "@probe.gl/stats@3.4.0", "@probe.gl/stats@^3.4.0":
```

Comment:
Can you dedupe the dependencies to have only one copy of each?

Reply:
You mean change in probe.gl to use `^` instead of pinning the dependencies: https://github.com/uber-web/probe.gl/blob/master/modules/log/package.json#L31 ?

Reply:
No. There are dependencies (likely luma or loaders) that specify probe with a `^` and once resolved to 3.4.0. Upgrading deck's dependency in submodule packages does not flush these in the lock file. You can see it by running `yarn list "@probe.gl/*". You can fix it by `yarn upgrade-interactive`.

Reply:
Made several attempts, but running yarn restores it. I am bumping dependencies, maybe that will fix it.